plugins/generator-1.15.2/forge-1.15.2/potioneffect.definition.yaml

```diff
@@ -0,0 +1,7 @@
+templates:
```

Comment:
You shouldn't change the mod element id (serialized name) of the custom potion effect because old workspaces will break if you change `potion` by `potioneffect` or `effect`. Also, in the current state, custom effects simply won't work as the serialized name is also the name of the definition name (`potion` != `potioneffect`).